vm/src/stdlib/io.rs

```diff
@@ -33,16 +35,39 @@ fn compute_c_flag(mode: &str) -> u16 {
     }
 }
 
-fn string_io_init(vm: &mut VirtualMachine, _args: PyFuncArgs) -> PyResult {
-    // arg_check!(vm, args, required = [(s, Some(vm.ctx.str_type()))]);
-    // TODO
-    Ok(vm.get_none())
+#[derive(Debug)]
+struct PyStringIO {
```

Comment:
For types in stdlib, I'd prefer we leave off the `Py` prefixes. To me, the purpose of those in the builtin types is to distinguish them from rust types, e.g. `PyString` vs `String`

Reply:
At first sight, I tend to disagree, my reason for prefixing all python types with `Py` would be to make clear that those are python types. Unlike for example VirtualMachine or Context. So for me its more than distuingish them from builtin rust types. That being said, I'm fine with both :)